app/components/layer-advanced/component.js

```diff
@@ -0,0 +1,28 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+
+  didRender() {
+    this._super(...arguments); 
+    this.$('div#froala-editor').html(this.get('layer.content'));
+    if(this.get("editMode") && this.get('showSettings') == false){
```

Comment:
Does this work too? ```!this.get('showSettings')``` instead of ```this.get('showSettings') == false)```

Also if you will use the == check you should use the strict check === 
https://github.com/airbnb/javascript#comparison-operators--equality